Makefile

```diff
@@ -0,0 +1,22 @@
+
+
+BUILD_DIR="./build"
+
+init:
+	# Run python script to build vocabulary with vectors.
+	# Adds custom tokeniser to base model.
+	echo "Not implemented"
+
+parser:
+	# Takes in a model output by init and adds "tagger" and "parser" pipelines.
+	echo "Not implemented"
+
+ner:
+	# Takes in a model output by init or parser and adds an "ner" pipeline.
+	echo "Not implemented"
+
+package:
+	# Create model packages for 1) The library and 2) The Spacy model.
+	bash scripts/create_model_package.sh ${BUILD_DIR}
```

Comment:
should this include the setup.py step and the pip install steps?

Reply:
No - I think we can assume that this will be run inside the docker image, or in an already setup environment. I could add a `make install` step?

Reply:
yeah i think having a make install step would be reasonable, just given that its a three step process. unless the intent is to make this an actual python package (ie put it on pypi)

Reply:
I added one and replaced what was in the Dockerfile with `make install` 👍  I think putting it on pip is not so hard, also.

Reply:
i think probably not, but downloading the models might be a barrier, not sure how much of a barrier though